osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/PermitsByThrottlingConfig.java

```diff
@@ -29,21 +30,21 @@ public class PermitsByThrottlingConfig {
       new ConcurrentHashMap<>();
 
   private final ThrottlingConfigRepository throttlingConfigRepository;
-  private final PermitRepository permitRepository;
-  private final PermitReleasedNotifier permitReleasedNotifier;
+  private final PermitService permitRepository;
```

Comment:
rename to permitService instead of permitRepository

Reply:
done